commands/weather/weather.js

```diff
@@ -6,7 +6,7 @@ const geocodeEndpoint = 'https://maps.googleapis.com/maps/api/geocode/json?key=g
 const darkskyEndpoint = 'https://api.darksky.net/forecast/key/lat,lng';
 
 function fahrenheitToCelcius(degree) {
-  return ((degree - 32) * 5 / 9).toFixed(0);
+  return (((degree - 32) * 5) / 9).toFixed(0);
```

Comment:
I guess putting the parentheses around `5/9` wouldn't look so "Lispy", but meh, the formula is so simple anyway. Turning off the rule for this line would also be OK.